### PR TITLE
docs(quickstart-and-gatekeepers): improved quickstart and gatekeepers

### PR DESCRIPTION
**Pull request by crisgarner**
# Description

Improved quickstart and gatekeepers

## Confirmation

- [x] I have read and understand MACI's [contributor guidelines](https://maci.pse.dev/docs/contributing) and [code of conduct](https://maci.pse.dev/docs/contributing/code-of-conduct).
- [x] I have read and understand MACI's [GitHub processes](https://github.com/privacy-scaling-explorations/maci/discussions/847).
- [x] I have read and understand MACI's [testing guide](https://maci.pse.dev/docs/testing).

### CODE REVIEW

File: website/versioned_docs/version-v2.0_alpha/quick-start/deployment.md [hunk 1]
```diff
@@ -34,6 +34,12 @@ For testing purposes, you can use the test zkeys and artifacts that you can down
 
 Please do not use test artifacts in production. If you do require zKeys configured for larger param sizes, please reach out to us if you will be using them in production and we'll discuss running a new ceremony for those parameters. To build new circuits artifacts for testing purposes, please refer to the [installation page](/docs/quick-start/installation#configure-circomkit) and to the [circuits](/docs/developers-references/zk-snark-circuits/introduction) section.
 
+### Note on Gatekeepers
+
+MACI uses a "gatekeeper" contract to configure and enforce the eligibility criteria of voters who can participate in MACI polls. In other words, it is a way to allowlist signups to the system. Operators of MACI can use the gatekeeper contract to configure signup eligibility and to protect against sybil attacks in their polls.
+
+It's necessary to define which gatekeeper are you going to use before deploying, please refer to the [gatekeepers page](/docs/developers-references/smart-contracts/Gatekeepers) section for more information on the supported Gatekeepers.
```
Comment (ctrlc03) [inline website/versioned_docs/version-v2.0_alpha/quick-start/deployment.md:41]: nit: "you are going"

Comment (ctrlc03) [inline website/versioned_docs/version-v2.0_alpha/quick-start/deployment.md:41]: nit: remove either page or section after gatekeepers

File: website/versioned_docs/version-v2.0_alpha/quick-start/poll-finalization.md [hunk 2]
```diff
@@ -15,16 +15,13 @@ This could be done using `maci-cli` or by running commands in the `contracts` fo
...
     --tally-file ../cli/tally.json \
     --output-dir ../cli/proofs/ \
-    --tally-wasm ../cli/zkeys/TallyVotes_10-1-2_test/TallyVotes_10-1-2_test_js/TallyVotes_10-1-2_test.wasm \
-    --process-wasm ../cli/zkeys/ProcessMessages_10-2-1-2_test/ProcessMessages_10-2-1-2_test_js/ProcessMessages_10-2-1-2_test.wasm
+    --start-block 12946802
 ```
 
 :::info
-If the poll was configured to use quadratic voting, please ensure your prove command has the flag `--use-quadratic-voting`. Omitting the flag will make it work with non quadratic voting only.
+If the poll was configured to use quadratic voting, please ensure your prove command has the flag `--use-quadratic-voting`. Omitting the flag will make it work with non quadratic voting only. We also suggest including the `--start-block` flag so the proving doesn't start from block 0.
```
Comment (ctrlc03) [inline website/versioned_docs/version-v2.0_alpha/quick-start/poll-finalization.md:24]: you could dig a bit deeper and explain that this part includes fetching all events from the smart contracts and starting from block zero would make it much longer and error prone due to rpc providers limitations

File: website/versioned_docs/version-v2.0_alpha/developers-references/smart-contracts/Gatekeepers.md [hunk 3]
```diff
@@ -16,6 +16,7 @@ The MACI repository comes with several out-of-the-box options:
 - `EASGatekeeper` - This restricts signups only to users who have received an attestation of a specific schema from a trusted attester.
 - `HatsGatekeeper` - This allows gatekeeping signups to only users who have a specific [Hat](https://www.hatsprotocol.xyz/).
 - `GitcoinPassportGatekeeper` - This allows gatekeeping signups to only users who have a specific [Gitcoin Passport](https://passport.gitcoin.co/) score.
+- `ZupassGatekeeper` - This allows gatekeeping signups only to users who have a valid Zupass PCD ticket.
```
Comment (ctrlc03) [inline website/versioned_docs/version-v2.0_alpha/developers-references/smart-contracts/Gatekeepers.md:19]: add a link to zupass PCD

**Comment by crisgarner** [comment]
Added the recommendations! thanks

File: website/versioned_docs/version-v2.0_alpha/quick-start/poll-finalization.md [hunk 4]
```diff
@@ -15,16 +15,13 @@ This could be done using `maci-cli` or by running commands in the `contracts` fo
...
     --tally-file ../cli/tally.json \
     --output-dir ../cli/proofs/ \
-    --tally-wasm ../cli/zkeys/TallyVotes_10-1-2_test/TallyVotes_10-1-2_test_js/TallyVotes_10-1-2_test.wasm \
-    --process-wasm ../cli/zkeys/ProcessMessages_10-2-1-2_test/ProcessMessages_10-2-1-2_test_js/ProcessMessages_10-2-1-2_test.wasm
+    --start-block 12946802
 ```
 
 :::info
-If the poll was configured to use quadratic voting, please ensure your prove command has the flag `--use-quadratic-voting`. Omitting the flag will make it work with non quadratic voting only.
+If the poll was configured to use quadratic voting, please ensure your prove command has the flag `--use-quadratic-voting`. Omitting the flag will make it work with non quadratic voting only. We also suggest including the `--start-block` flag, proving requires fetching all events from the smartcontract and by default starts from block zero, this would take a lot of time and is error-prone due to RPC provider limitations.
```
Comment (ctrlc03) [inline website/versioned_docs/version-v2.0_alpha/quick-start/poll-finalization.md:24]: nit: smart contracts

**Review by ctrlc03 — APPROVED** [review]
Left one more comment, after that feel free to merge thanks

**Review by kittybest — APPROVED** [review]
thanks!

**Pull request merged**